src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/DeferredProjectsUtility/DeferredProjectVSUtility.cs

```diff
@@ -0,0 +1,109 @@
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.VisualStudio;
+using Microsoft.VisualStudio.Shell;
+using Microsoft.VisualStudio.Shell.Interop;
+using NuGet.VisualStudio;
+
+namespace NuGet.PackageManagement.VisualStudio
+{
+    public static class DeferredProjectVSUtility
```

Comment:
I wouldn't have mind adding these 2-3 apis in 'VSSolutionManager' itself since this solution is going to be short lived and only applicable for 15.3-rtm